filers/default/src/main/java/org/commonjava/indy/filer/def/conf/DefaultStorageProviderConfiguration.java

```diff
@@ -120,6 +120,8 @@ public Properties getSystemPropertyAdditions()
 
     private String fileChecksumAlgorithm = "SHA-256";
 
+    private String deduplicatePattern = "^(generic|npm).+";
```

Comment:
Seems not used?

Reply:
Used in getDeduplicatePattern().

Reply:
And the getDeduplicatePattern() is used in DefaultGalleyStorageProvider.

Reply:
Ok. Seems it is like some type of constant but used lower case, so makes me confused.

Reply:
Yep. It only appears once here so I am lazy to create a CONSTANT for it :)